libknet/libknet.h

```diff
@@ -77,6 +77,7 @@ typedef uint16_t knet_node_id_t;
  * Where possible, set traffic priority to high.
  * On Linux this sets the TOS to INTERACTIVE (6),
  * see tc-prio(8) for more infomation
+ * A dscp value may be configured, see knet_handle_setprio_dscp.
```

Comment:
might be worth specifying in the documentation that this value must be set before configuring knet links.